libs/wire-api/src/Wire/API/Conversation.hs

```diff
@@ -175,37 +176,68 @@ accessRolesSchemaOpt = toOutput .= accessRolesSchemaTuple `withParser` validate
 accessRolesSchemaTuple =
   (,)
-    <$> fst .= optFieldWithDocModifier "access_role" (description ?~ "Deprecated, please use access_role_v2") (maybeWithDefault A.Null schema)
-    <*> snd .= optFieldWithDocModifier "access_role_v2" (description ?~ desc) (maybeWithDefault A.Null $ set schema)
+    <$> fst
+    .= optFieldWithDocModifier "access_role" (description ?~ "Deprecated, please use access_role_v2") (maybeWithDefault A.Null schema)
+    <*> snd
+    .= optFieldWithDocModifier "access_role_v2" (description ?~ desc) (maybeWithDefault A.Null $ set schema)
   where
     desc = "This field is optional. If it is not present, the default will be `[team_member, non_team_member, service]`. Please note that an empty list is not allowed when creating a new conversation."
 
 conversationMetadataObjectSchema :: ObjectSchema SwaggerDoc ConversationMetadata
 conversationMetadataObjectSchema =
   ConversationMetadata
-    <$> cnvmType .= field "type" schema
+    <$> cnvmType
+    .= field "type" schema
     <*> cnvmCreator
-      .= fieldWithDocModifier
-        "creator"
-        (description ?~ "The creator's user ID")
-        schema
-    <*> cnvmAccess .= field "access" (array schema)
-    <*> cnvmAccessRoles .= accessRolesSchema
-    <*> cnvmName .= optField "name" (maybeWithDefault A.Null schema)
-    <* const ("0.0" :: Text) .= optional (field "last_event" schema)
+    .= fieldWithDocModifier
+      "creator"
+      (description ?~ "The creator's user ID")
+      schema
+    <*> cnvmAccess
+    .= field "access" (array schema)
+    <*> cnvmAccessRoles
+    .= accessRolesSchema
+    <*> cnvmName
+    .= optField "name" (maybeWithDefault A.Null schema)
+    <* const ("0.0" :: Text)
+    .= optional (field "last_event" schema)
     <* const ("1970-01-01T00:00:00.000Z" :: Text)
-      .= optional (field "last_event_time" schema)
-    <*> cnvmTeam .= optField "team" (maybeWithDefault A.Null schema)
+    .= optional (field "last_event_time" schema)
+    <*> cnvmTeam
+    .= optField "team" (maybeWithDefault A.Null schema)
     <*> cnvmMessageTimer
-      .= optFieldWithDocModifier
-        "message_timer"
-        (description ?~ "Per-conversation message timer (can be null)")
-        (maybeWithDefault A.Null schema)
-    <*> cnvmReceiptMode .= optField "receipt_mode" (maybeWithDefault A.Null schema)
+    .= optFieldWithDocModifier
+      "message_timer"
+      (description ?~ "Per-conversation message timer (can be null)")
+      (maybeWithDefault A.Null schema)
+    <*> cnvmReceiptMode
+    .= optField "receipt_mode" (maybeWithDefault A.Null schema)
 
 instance ToSchema ConversationMetadata where
   schema = object "ConversationMetadata" conversationMetadataObjectSchema
 
+-- | Public-facing global team conversation.
+-- Membership is implicit. Every member of a team is part of it.
+-- Protocol is also implicit: it's always MLS.
+data GlobalTeamConversation = GlobalTeamConversation
```

Comment:
Now that I think about it, if we're to stay consistent with how we've implemented MLS so far, this should go somewhere under `Wire.API.MLS`, perhaps in a new module.